src/DynamoCoreWpf/Views/GuidedTour/RealTimeInfoWindow.xaml.cs

```diff
@@ -15,6 +17,31 @@ public partial class RealTimeInfoWindow : Popup
         /// </summary>
         public string TextContent { get; set; }
 
+        /// <summary>
+        /// This property indicates if the Hyperlink will be shown in the RealTimeInfo window
+        /// </summary>
+        public bool ShowHyperlink { get; set; }
+
+        /// <summary>
+        /// This property indicates if the Header will be shown in the RealTimeInfo window
+        /// </summary>
+        public bool ShowHeader { get; set; }
+
+        /// <summary>
+        /// This property contains the text that will be shown in the Hyperlink
+        /// </summary>
+        public string HyperlinkText { get; set; }
+
+        /// <summary>
+        /// This property contains the text that will be shown in the Header
+        /// </summary>
+        public string HeaderContent { get; set; }
+
+        /// <summary>
+        /// This property contains the URI that will be opened when the Hyperlink is clicked
+        /// </summary>
+        public Uri HyperlinkUri { get; set; }
+
```

Comment:
`RealTimeInfoWindow.ShowHyperlink` field can be removed, if `HyperlinkUri` is a non-empty string, then it can show the hyperlink. Same for `RealTimeInfoWindow.ShowHeader` field, which can be inferred by `HeaderContent` not being an empty string.